packages/documentation-v7/src/stories/components/range/range.stories.ts

```diff
@@ -0,0 +1,288 @@
+        arg: 'useBoundaries',
+      },
+      control: {
+        type: 'number',
+        step: 1,
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    showValue: {
+      name: 'Show Value',
+      description: 'Render the component with or without a visible value.',
+      control: {
+        type: 'radio',
+        labels: {
+          none: 'None',
+          text: 'Text',
+          input: 'Input',
+        },
+      },
+      options: ['none', 'text', 'input'],
+      table: {
+        category: 'General',
+      },
+    },
+    disabled: {
+      name: 'Disabled',
+      description:
+        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<span className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</span>',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'States',
+      },
+    },
+    validation: {
+      name: 'Validation',
+      description: "Controls the display of the component's validation state.",
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'is-valid': 'Valid',
+          'is-invalid': 'Invalid',
+        },
+      },
+      options: ['null', 'is-valid', 'is-invalid'],
+      table: {
+        category: 'States',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function render(args: Args, context: StoryContext) {
+  const [_, updateArgs] = useArgs();
+
+  const id = `${context.viewMode}_${context.name.replace(/\s/g, '-')}_ExampleRange`;
+  const classes = ['form-range', args.validation].filter(c => c && c !== 'null').join(' ');
+
+  const useAriaLabel = args.hiddenLabel;
+  const label: TemplateResult | null = !useAriaLabel
+    ? html`
+        <label class="form-label" for="${id}">${args.label}</label>
+      `
+    : null;
+
+  const contextual: (TemplateResult | null)[] = [
+    args.validation === 'is-valid'
+      ? html`
+          <p class="valid-feedback">Ggranda sukceso!</p>
+        `
+      : null,
+    args.validation === 'is-invalid'
+      ? html`
+          <p class="invalid-feedback">Eraro okazis!</p>
+        `
+      : null,
+  ];
+
+  const control: TemplateResult = html`
+    <input
+      id="${id}"
+      class="${classes}"
+      type="range"
+      value="${args.value}"
+      .valueAsNumber="${args.value}"
+      min="${args.useBoundaries ? args.min : nothing}"
+      max="${args.useBoundaries ? args.max : nothing}"
+      step="${args.useBoundaries ? args.step : nothing}"
+      ?disabled="${args.disabled}"
+      aria-label="${useAriaLabel ? args.label : nothing}"
+      ?aria-invalid="${VALIDATION_STATE_MAP[args.validation]}"
+      @change="${(e: MouseEvent) => updateArgs({ value: (e.target as HTMLInputElement).value })}"
```

Comment:
Provides more responsive feedback.

```suggestion
      @input="${(e: MouseEvent) => updateArgs({ value: (e.target as HTMLInputElement).value })}"
```